### DISCUSSION TITLE
x-reactions 导致页面崩溃。。。

### DISCUSSION THREAD

**Discussion by TheKenS** [alibaba/formily#3393] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/49146439/190055008-9c12e2e5-7225-4364-a347-46621ecc594b.png)
<img width="763" alt="image" src="https://user-images.githubusercontent.com/49146439/190055068-cfd30d03-4567-4475-bad6-1c43aaeb1d85.png">

**Comment by janryWang** [comment]
给个demo吧

**Comment by PluckySaltyfish** [comment]
找到原因了吗
